README.md

```diff
@@ -160,6 +160,40 @@ addServiceWorker({
 ```
 
 
+### Generating a Push Manifest
+
+Polymer-Build can automatically generate a [push manifest](https://github.com/GoogleChrome/http2-push-manifest) for your application. This JSON file can be read by your http2 push-enabled web server to more easily construct the appropriate `Link: <URL>; rel=preload; as=<TYPE>` headers(s) for http2 push/preload.
+
+> **Note:** Push manifests can only be generated for app-shell projects (projects that define a shell and fragments). The manifest describes the following behavior: Requesting the shell should push any shell dependencies as well. Requesting a fragment should push any dependencies of that fragment *that were not already pushed by the shell.*
```

Comment:
So in that example, all dependencies are pushed when `/index.html` is requested, but then the service worker caches `index.html` and intercepts future requests to the server that would repush the entire app again.

Okay, if the user is asking for a push manifest in their non-app-shell application that makes sense. +1 will add.

This also reminds me that I should add a warning if `--add-push-manifest` is included but `--add-service-worker` is not, since right now the entire pushing strategy relies on the server worker intercepting unnecessary requests.